config/v1/types_infrastructure.go

```diff
@@ -505,9 +505,113 @@ type BareMetalPlatformStatus struct {
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
 }
 
+type OpenStackAPIBGPPeer struct {
+	// asn is the ASN of the peer.
```

Comment:
What's an ASN? Is this a common abbreviation or should we spell it out? What is the format of an ASN?

Reply:
spelled out `Autonomous System number` in the godoc